compiler/crates/relay-typegen/tests/generate_flow/fixtures/typename-on-union.expected

```diff
@@ -185,14 +166,25 @@ import type { FragmentReference } from "relay-runtime";
 declare export opaque type TypenameOutsideWithAbstractType$ref: FragmentReference;
 declare export opaque type TypenameOutsideWithAbstractType$fragmentType: TypenameOutsideWithAbstractType$ref;
 export type TypenameOutsideWithAbstractType = {|
-  +__typename: string,
+  +__typename: "User",
+  +firstName: ?string,
+  +address: ?{|
+    +street: ?string,
+    +city: ?string,
+  |},
+  +username?: ?string,
+  +address?: ?{|
```

Comment:
hmm, why `address` is included twice here?

Reply:
Fixed!